src/app/store/auth.slice.test.ts

```diff
@@ -78,6 +86,73 @@ describe('authSlice', () => {
       });
     });
   });
+
+  describe('createPasswordSuccess', () => {
+    it('sets newUser to false and loggedIn to true', () => {
+      expect(
+        slice.reducer({ newUser: true, loggedIn: false }, createPasswordSuccess())
+      ).toStrictEqual({
+        newUser: false,
+        loggedIn: true,
+        error: undefined
+      });
+    });
+
+    it('clears the error', () => {
+      expect(
+        slice.reducer({ newUser: true, loggedIn: false, error: 'foo bar' }, createPasswordSuccess())
+      ).toStrictEqual({
+        newUser: false,
+        loggedIn: true,
+        error: undefined
+      });
+    });
+  });
+
+  describe('createPasswordFailed', () => {
+    it('sets error to the payload', () => {
+      const state = { newUser: true, loggedIn: false };
+      expect(slice.reducer(state, createPasswordFailed('Foo bar'))).toStrictEqual({
+        ...state,
+        error: 'Foo bar'
+      });
+    });
+  });
+});
+
+describe('createPasswordWorker', () => {
+  it('dispatches createPasswordSuccess on success', () => {
+    return expectSaga(createPasswordWorker, createPassword('foobar'))
+      .provide([[call.fn(ipcBridgeRenderer.db.invoke), true]])
+      .call(ipcBridgeRenderer.db.invoke, {
+        type: DBRequestType.INIT,
+        password: 'foobar'
+      })
+      .put(createPasswordSuccess())
+      .silentRun();
+  });
+
+  it('navigates to the dashboard on succesful login', () => {
+    return expectSaga(createPasswordWorker, createPassword('foobar'))
+      .provide([[call.fn(ipcBridgeRenderer.db.invoke), true]])
+      .call(ipcBridgeRenderer.db.invoke, {
+        type: DBRequestType.INIT,
+        password: 'foobar'
+      })
+      .put(push(ROUTE_PATHS.HOME))
+      .silentRun();
+  });
+
+  it('dispatches createPasswordFailed on error', () => {
+    return expectSaga(createPasswordWorker, createPassword('foobar'))
+      .provide([[call.fn(ipcBridgeRenderer.db.invoke), false]])
+      .call(ipcBridgeRenderer.db.invoke, {
+        type: DBRequestType.INIT,
+        password: 'foobar'
+      })
+      .put(createPasswordFailed(translateRaw('LOGIN_ERROR')))
```

Comment:
Can we use another string for this so it doesn't get confusing when we want to add better copy?